introduction/java_loops_i/Solution.java

```diff
@@ -0,0 +1,13 @@
+import java.util.Scanner;
+
+public class Solution {
+  private static final int NUMBER_OF_ITERATIONS = 10;
+  public static void main (String[] args) {
+    Scanner scan = new Scanner(System.in);
+    int inputNumber = scan.nextInt();
+    for (int iteration = 1; iteration <= NUMBER_OF_ITERATIONS; iteration++) {
+      int resultOfMultiplication = inputNumber * iteration;
+      System.out.println(inputNumber + " x " + iteration + " = " + resultOfMultiplication);
```

Comment:
Alternatively you could use `String.format()`. It's not a requirement, just a matter of preference :)